test/blackbox-tests/test-cases/enabled_if/eif-arch_sixtyfour.t/run.t

```diff
@@ -0,0 +1,31 @@
+Testing %{arch_sixtyfour} in enabled_if
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.9)
+  > EOF
+
+  $ dune exec -- ./hello.exe
+  Hello, World!
+
+Testing the version guard
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.8)
+  > EOF
+
+  $ dune exec -- ./hello.exe
+  File "dune", line 4, characters 2-41:
+  4 |   (= %{arch_sixtyfour} %{arch_sixtyfour})))
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Error: Unexpected list
+  [1]
+
+The error message is not ideal. It should say something like this instead:
+
+File "dune", line 3, characters 13-30:
+3 |  (enabled_if %{arch_sixtyfour}))
+Error: %{arch_sixtyfour} is only available since version 3.9 of the dune
```

Comment:
I can't reproduce the failure in the error message. It's outputting the right one here (and CI agrees).

Reply:
Weird this is what I get on this branch.

Reply:
64